### PR TITLE
Expand to full citation

### PR DESCRIPTION
**Pull request by hosking**

### CODE REVIEW

File: _data/Achievement.yaml [hunk 1]
```diff
@@ -3,6 +3,39 @@
...
+    reduce bias and improve scientific quality.  McKinley also played
+    a key role in advancing women and minorities in programming
+    languages, and more generally, within computer science. She
+    energetically championed this cause and was a dedicated mentor to
+    her 23 PhD students and countless other researchers.
+
+    In short, Kathryn McKinley is the "whole picture" of what the
+    SIGPLAN achievement award seeks to recognize.
+
+Kathryn McKinley has made seminal contributions to a number of
```
Comment (sampsyo) [inline _data/Achievement.yaml:38]: It looks like maybe the old version of the citation is still here, and needs to be deleted?

**Comment by hosking** [comment]
Fixed inadvertent inclusion of old abbreviated citation.

File: _data/Achievement.yaml (hunk 1 above)
Comment (hosking) [reply _data/Achievement.yaml]: Oops, fixed.

**Comment by sampsyo** [comment]
Excellent; thanks!!

**Pull request merged**